tests/ruff.toml

```diff
@@ -10,4 +10,5 @@ extend-ignore = [
     "E731", # lambda assignment
     "F841", # unused local variable
     "S101", # asserts allowed in tests
+    "NPY201", # numpy 2.* compatibility check
```

Comment:
snuck in from another PR?

Reply:
Actually no, this got flagged (`np.trapz` usage) and I disabled it. We should enable it again once we support numpy 2.